### DISCUSSION TITLE
Does command "jbrowse text-index" support GTF annotation file?

### DISCUSSION THREAD

**Discussion by luoxun-xl** [GMOD/jbrowse-components#3489] [General] [answered]
Hi, I use JBrowse2 (v2.3.3) to visualize genomic data, and I could add genome annotation tracks in both GFF and GTF format. The GFF and GTF format track could be visualized normally in JBrowse2. However, it seemed that the command [`jbrowse text-index`](https://jbrowse.org/jb2/docs/cli/#jbrowse-text-index) can process the GFF format track well but does not support the GTF format. 

Except for transforming GTF to GFF format, is there any other solution to make a text-index for the GTF format track?

**Comment by cmdcolin** [comment, marked as answer]
Currently, you would need to convert GTF->GFF for the text-index. There was some work to index GTF with text-index but wasn't completed yet

**Reply by luoxun-xl** [reply]
Thanks! Hope it will be updated in future version.
